metricbeat/Makefile

```diff
@@ -25,7 +25,7 @@ kibana:
 collect-docs: python-env
 	@rm -rf docs/modules
 	@mkdir -p docs/modules
```

Comment:
Should `CollectDocs` take care of removing and recreating the directory too? Other mage targets we have remove the target directories before recreating the files.

Reply:
Ah, that's a good idea.